drivers/soundwire/cadence_master.c

```diff
@@ -881,13 +879,15 @@ irqreturn_t sdw_cdns_irq(int irq, void *dev_id)
 		return IRQ_NONE;
 
 	if (int_status & CDNS_MCP_INT_RX_WL) {
+		struct sdw_bus *bus = &cdns->bus;
+		struct sdw_defer *defer = &bus->defer_msg;
+
```

Comment:
we don't need define variable bus here,  use "struct sdw_defer *defer = cdns->bus.defer_msg;" ?

Reply:
I like keeping indirections simple enough, and specifically make it clear that the defer stuff is a BUS structure.